backends/xnnpack/test/ops/test_exp.py

```diff
@@ -34,6 +34,9 @@ def run_exp_test(self, inputs):
             .run_method_and_compare_outputs()
         )
 
+    # TODO (leafs1): Fix flaky tests. Land fix asap
+    # and cherry-pick onto release/0.7 branch
+    @unittest.skip(reason="For float16, numerical discepancies are too high")
```

Comment:
@mcr229 should we increase atol/rtol for this, or normalize/range-select inputs?